build.gradle

```diff
@@ -38,14 +38,14 @@ buildscript {
 
         okhttp_version = '3.7.0'
 
-        permission_dispatcher_version = '4.3.0'
         play_service_version = '12.0.1'
 
         retrofit_version = '2.4.0'
         rxandroid_version = '2.0.1'
         rxbinding_version = "3.1.0"
         rxjava_version = '2.1.4'
         rxkotlin_version = '2.2.0'
+        rxpermission_version = "0.10.2"
```

Comment:
@luongvo should we update to latest version `0.12` [Rxpermission](https://github.com/tbruyelle/RxPermissions)

Reply:
@manh-t Rxpermission 0.12 requires rxjava3 as we're using rxjava2, it should be upgraded later